### DISCUSSION TITLE
Can I use DOCX to open an existing .docx file?

### DISCUSSION THREAD

**Discussion by gilbriatore** [dolanmiu/docx#2109] [General]
I need to open some .docx files, modify someone, and merge others... can I use DOCX to open existing files? If so, are there some examples of how I could do this?

**Comment by dolanmiu** [comment]
Yes, this is possible, 

Something like this?

https://github.com/dolanmiu/docx/blob/master/demo/85-template-document.ts

**Reply by gilbriatore** [reply]
Thank you @dolanmiu !
